### DISCUSSION TITLE
Does stryker-net support collecting test coverage for mutators?

### DISCUSSION THREAD

**Discussion by Rayffer** [stryker-mutator/stryker-net#2106] [General]
I've looked through the documentation for stryker-net and I've not been able to find how to enable an option for collecting how much mutators each tests covers/kills, like in stryker-js.

![image](https://user-images.githubusercontent.com/34773204/178241329-990bdb70-7160-4523-8eb1-4ed702e10c56.png)

Is it implemented as of today in .net? If so, what configuration parameters are needed in order to produce said test coverage report.

**Comment by rouke-broersma** [comment]
This is being implemented #1850
